projects/1.18-fabric/assets/incantationem/incantationem/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "enchantment.incantationem.bunnys_hop": "兔跳",
+  "enchantment.incantationem.swine_bane": "猪类杀手",
+  "enchantment.incantationem.charmed": "庇佑",
+  "enchantment.incantationem.decay": "破败",
+  "enchantment.incantationem.forging_touch": "煅烧之触",
+  "enchantment.incantationem.last_stand": "背水一战",
+  "enchantment.incantationem.magnetic": "磁性",
+  "enchantment.incantationem.reaping_rod": "收割之竿",
+  "enchantment.incantationem.retreat": "撤退",
+  "enchantment.incantationem.venomous": "淬毒",
+
+  "enchantment.incantationem.recklessness": "鲁莽诅咒",
+  "enchantment.incantationem.thunder": "引雷诅咒",
+  "enchantment.incantationem.tough_luck": "不幸诅咒",
+
+
+  "message.incantationem.decay.applied": "*已施加破败效果！*",
+  "message.incantationem.forging_touch.applied": "*已施加煅烧之触效果！*",
+  "message.incantationem.last_stand.applied": "*已施加背水一战效果！*",
+  "message.incantationem.reaping_rod.applied": "*已施加收割之竿效果！*",
+  "message.incantationem.retreat.applied": "*已施加撤退效果！*",
+  "message.incantationem.venomous.applied": "*已施加淬毒效果！*",
+
+  "message.incantationem.thunder.applied": "*已施加引雷诅咒效果！:(*",
+
+  "enchantment.incantationem.bunnys_hop.desc": "站在泥土/草方块类方块上提供跳跃提升效果。",
+  "enchantment.incantationem.swine_bane.desc": "提高对猪类生物的伤害，例如猪，猪灵，疣猪兽等。",
+  "enchantment.incantationem.charmed.desc": "提高获得高质量战利品的概率和触发本模组增益效果的概率，降低本模组诅咒触发的概率。",
+  "enchantment.incantationem.decay.desc": "攻击有一定概率造成凋零效果。",
+  "enchantment.incantationem.forging_touch.desc": "破坏方块有一定概率掉落其烧制后的产物。（比如：沙子->玻璃）",
+  "enchantment.incantationem.last_stand.desc": "低生命值时受到攻击有概率获得力量效果。",
+  "enchantment.incantationem.magnetic.desc": "吸引玩家周围的物品和经验球。",
+  "enchantment.incantationem.reaping_rod.desc": "用鱼竿类物品击杀敌对生物时给予类似附魔金苹果的效果。",

```

被`给予～效果`的客体是？玩家、鱼竿、还是被击杀的生物？
原文没提到“鱼竿”，此外“鱼竿类物品”可以统称“鱼竿”
```suggestion
  "enchantment.incantationem.reaping_rod.desc": "用鱼竿击杀敌对生物时给予玩家类似附魔金苹果的效果。",
```
```suggestion
  "enchantment.incantationem.reaping_rod.desc": "用鱼竿击杀敌对生物时获得类似附魔金苹果的效果。",
```
此外上文都是`获得`，到这里变成了`给予`